pkg/cache/v3/linear.go

```diff
@@ -71,22 +68,25 @@ func (c *cachedResource) getVersion(useStableVersion bool) (string, error) {
 	return c.getStableVersion()
 }
 
-type watches struct {
-	// sotw keeps track of current sotw watches, indexed per watch id.
-	sotw map[uint64]ResponseWatch
-	// delta keeps track of current delta watches, indexed per watch id.
-	delta map[uint64]DeltaResponseWatch
-}
+type watch interface {
+	// Only used for statistics
```

Comment:
From https://github.com/DataDog/go-control-plane/pull/14/files#diff-81a59a320bd52918b18daec36cb809a405dc05844fbf9507d8126d64b03ad594R299 it looks like it's not true?